doc/developer/building-frr-on-ubuntu1804.rst

```diff
@@ -63,50 +64,50 @@ Add frr groups and user
 
 ::
 
-    sudo groupadd -g 92 frr
-    sudo groupadd -r -g 85 frrvty
-    sudo adduser --system --ingroup frr --home /var/run/frr/ \
-       --gecos "FRR suite" --shell /sbin/nologin frr
-    sudo usermod -a -G frrvty frr
+   sudo groupadd -g 92 frr
```

Comment:
Why is this `groupadd` not adding a system group? (it is missing the `-r`)

Reply:
Agreed we should definitely be doing this :(